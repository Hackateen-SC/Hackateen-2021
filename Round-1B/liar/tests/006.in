700
487390648 396789423 753706676 807750580 460540003 60368219 866422286 688993548 39837215 237700416 150359302 676053993 832366914 78322250 648806750 111597214 601442688 628667394 264352624 742549164 702469495 28924401 121907525 222270156 266153405 345998253 971704498 430702164 792059520 926230066 505929182 79089796 351792354 994695495 642236357 180533974 453341104 824922296 108886111 407134395 407270277 58188688 980860411 947006468 554327559 381314214 195228414 668034920 445295645 714982910 984377825 580074200 531298562 927812060 770691690 878553915 891901177 144893096 354902769 829487145 737346955 330995459 901381192 759102354 282662191 818370341 875233568 972207925 269333830 924450355 403252868 376261441 459903894 392943601 573701745 639378470 22110728 986859992 811343434 382142895 799252654 444158927 314635407 714883130 721352150 122078974 461672062 968310982 661957343 534647976 334524401 604863335 316028230 466067166 340746229 764005011 153083220 70005401 477496205 585431534 680775499 371578232 591777858 273834090 799812139 956015306 971201947 885854578 649751668 849854050 550727758 365608496 201356515 920026666 112824570 100866277 98669817 328039615 81564194 41442691 528906630 862477601 293197746 530192074 877032381 683423167 372144645 610813117 705408450 154229432 139846877 629494653 325853862 899862420 370047670 573063308 410505441 671529681 167506673 305491278 959122050 703409682 126221340 548154319 792279850 910807506 928433810 350220709 39985233 705853751 12311645 351788404 650481767 308222129 680781453 797150971 14132094 369054338 286894169 269615864 628664393 203399617 662839199 764988844 246277614 465407895 111108766 882810841 746593515 1767477 426078521 716730918 858287692 151275738 130586612 59174383 88907798 205609113 870511896 23955194 838506365 790304669 183652177 124615610 87764366 601801554 435359138 242473374 844296317 865035165 573494588 960285420 626532931 142287179 305312622 835998964 391724246 483033513 52716430 620857141 667896178 533943170 622462544 872916292 558309575 310158569 992221549 69565790 468703902 626586194 626805692 53930244 847668986 557008049 676291846 276353364 453756624 383458861 517047841 381355796 750638452 464766424 632444943 886896859 115434750 179213826 501035875 505412921 139422882 894663888 403848742 779461810 80986662 56754846 15612480 268805457 630009805 604017674 620230766 841546197 911425073 302195086 262118577 586462017 253334544 322866517 617424238 563520104 995448676 425802581 413902436 490554397 104436971 377575087 374231509 274578245 260991195 663893707 970034362 367176278 790513002 708563237 194434568 197785664 555862468 362264877 65209585 456043987 266468224 123767056 53583831 983476280 189966382 4669923 34282360 69989702 936686119 645037820 17081929 951602029 919790787 636407408 20601431 953324478 24877626 547986124 517327290 972073604 810037949 965203802 724029715 738175651 624041756 384044453 6383734 204350949 444696993 820029278 343344450 790760491 381118432 229743500 676035810 708774219 89443177 284080196 246578524 944745045 531035867 758500649 587841614 725688898 272021760 351484810 466111689 494311454 80077579 233743963 919759809 876878588 379133681 367603423 114800187 227703154 980047284 164835087 361723867 146664073 975514504 945552815 447487424 383250910 822786538 772699205 468114273 801191370 408180971 306945583 771792320 885788919 442807132 235733332 987802484 54916131 441613121 867536281 490224305 363934939 89001669 429072846 988370792 669647353 775867829 772731834 540924076 995432745 302726868 403440063 967948384 574934577 561330180 843723474 446973689 318550513 682526422 802836346 949531053 873021317 872784142 966653682 238927199 920291745 555837237 434923157 917782865 644043841 422695172 606898793 769338766 508344733 379020942 143023139 448263675 646255606 424568124 228809833 330452222 197268281 456935714 254383825 35385653 686505209 484977178 145805320 248490873 935353280 639404829 647903554 14865326 803944766 241852175 256567598 470365603 378256864 620123666 368515855 484855385 229598991 407470655 706146229 556963969 669426200 757499598 299647687 308752316 380075794 998036786 584102758 15526203 823924922 573875058 17634072 966283682 580072579 978102967 622614337 165447966 702489328 989803926 651091724 189961938 508417029 359994848 352264819 85698967 901637190 507036533 51110426 702971218 474422410 675390094 279406808 121694868 672372612 960053286 790387723 529667403 321475150 401431562 866772355 723916701 287288121 229921397 164577469 47389408 842313022 239617236 228709149 51308409 980472942 516664489 101536589 758748838 124646645 202598998 377930364 458232466 330434634 656247380 355505279 857348071 240467301 163942995 316761901 345342699 292433284 198637416 118841372 276281698 952161704 263245875 733078720 424370637 679535233 691612245 842117040 769312012 320193277 468560162 785448283 67496875 804778038 503786975 71225600 308494595 805375237 416895190 361435160 911033001 889302750 695132903 923115030 405195624 849235252 267884911 541262261 179643459 632838758 727213507 122780885 474644257 240867690 287216119 753056298 700539887 811420438 114885144 463621215 686630155 539784793 952643203 17475910 643528668 866261720 700826391 352513209 42693243 232402635 10043290 853767437 42980615 623468097 234477321 679772446 594013424 918149150 620004801 409774011 746945873 530886711 104487639 448014545 541704108 314780977 63368476 207203142 574918136 838628412 795992777 291422936 725388597 960163498 698329063 683674878 513116546 650368296 833015116 150912358 961135376 773440461 331827304 792071238 203196628 847894320 903450566 993574570 519544946 802154304 403735109 270179539 85164708 423954439 394070758 520407218 108692648 843560854 376106194 22239842 635627273 689564269 632043410 573220157 987957299 28589125 801915587 602033422 406626222 213394899 868956959 919240269 374724270 882674318 869288621 742909365 994595105 244182466 679841941 715161770 276064047 577189106 111921242 226841842 963702411 879709521 91128161 4157510 846303796 640117166 150485934 974029829 127354368 818219561 384261317 483829902 277323721 349703267 480674256 129268480 664652253 998155116 860745190 220081162 264478169 39461486 132069615 515738376 144080823 206283381 942200411 30537260 429956904 495965968 797812661 284253450 553920477 132147052 9751672 974684280 160892685 549569114 779265988 456228464 826010204 231124640 58435987 355977904 631774795 706904338 536350466 465139883 848051621 487408508 42059558 471237124 895480754 290206504 701999246 9639988 237966611 729538741 357514526 308120527 272185228 12138331 152519146 468588411 751521504 838690962 932906078 647398844 7841569 840491386 497992937 392295008 304978996 656570435 173795807 525935792 391284463 826399432 772937311 463009541 27051426 616382281 38410135 350336422 400655894 471999427 616104915 984407746 956985548 355919331 607605420 99264388 658631030 567783844 834325322 884655644 608479626 256644551
LGGGLGGLGLLGGGGGLGGGGLLGGGGGLLLLGLLLGLLLGGGLGGLLGGGGLGLLGLLLGLLLGGLGGLGGLGLLLGGGLGGLLLGGGGLGLLGLLLGLGLLGLLGGLGGLLGGLGGGLGLLLGLLGLLGGLLGGLLLLLGLGLLGGGGGLLGLGGGGLLLLLGLLGLLLGLLGGGGGGGLGGLLGGLLGGGGLLLLGGLLGGLLLGLGLGLLGGGGLLLLLGLGLGGGGGGGLGGGLLLGLLLLLGGGLGGLLLLLGGLGLGLLLGGGGLGGLGGLLGLLGGGGGLLGGLGLLGLGGGLGGGLLLGGLGGLGGGLLLLGLGLGGLLLLLGLLLGLLGLLGLGGLLLLLGGGLGGGGLLGLLLGLLLGLGGLGGGGGLLLLGLGLGLGGGGGLGGGLLLLGLLLGGLLLGLLLGLLLGLGLLLLLGLLLGLGLGLLGLLLGLLGGGGLLGLGLGLLGGLGGLGLGGGGLLGGGGGLGLGGLLGLGGLLGGLLLLGLLLGLLLGGGLLLGGLGLGLGGGGLLGGGGLGGGLLLLLLGLLGGLGGLLLLGGGLGLGLGLLLGGGGLLLLGGGLGGGGLLLLGLGLLGLLGLLLGLLGLLGGLGGGLGLLGLLLGLLLGGLGGGLLGLGGLGGGGGLLGLGLGGLLGGGLLLLLGLLGGLLLGGGGLLLLGGGGLGLGGGLGLLGLLGGGLLLLGLLGLGGG
