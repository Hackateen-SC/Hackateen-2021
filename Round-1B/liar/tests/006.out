351484811 316
